x/tokenfactory/simulation/operations.go

```diff
@@ -0,0 +1,405 @@
+package simulation
```

Comment:
This completely replaces all functionality in (commented out) https://github.com/CosmWasm/token-factory/blob/main/x/tokenfactory/simulation/sim_msgs.go right?

If so, can you delete that file (I just left it for reference for whoever implemented sims)

Reply:
Got it!!!